packaging/sources/preupgrade-assistant-pykickstart.patch

```diff
@@ -1276,6 +1276,16 @@ index f71de9f..3c5b7d4 100644
          # excluded groups so whatever tool doing package/group installation can
          # take appropriate action.
          self.excludedGroupList.extend(excludedGroupList)
+@@ -664,9 +664,6 @@ class KickstartParser:
+ 
+             elif self._state == STATE_PACKAGES:
+                 if not args and self._includeDepth == 0:
+-                    if self.version >= F8 :
+-                        warnings.warn(_("%s does not end with %%end.  This syntax has been deprecated.  It may be removed from future releases, which will result in a fatal error from kickstart.  Please modify your kickstart file to use this updated syntax.") % "%packages", DeprecationWarning)
+-
```

Comment:
Rationale for this change:
**Remove warning of missing %end in kickstart**
    
    - RHEL 5 kickstart is quite often installed using
      kickstart with %end after %package
    - the generated kickstart for RHEL 7 is generated correctly
      with %end